.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+on:
```

Comment:
It looks like you don't have a name for this and the filename isn't great.

Reply:
@neomilium can you take a look at this?

Reply:
It seems I will not have so much spare time during next weeks. So, feel free to rework my PR if you want to merge it quickly.

Reply:
@ekohl What name you prefer `ci`, `Continuous integration`, `Tests`? Or maybe you have better suggestion. Thx

Reply:
I'm leaning to keeping it short and use `CI`.

Reply:
OK, so `CI` for the name but you point me that `ci.yml` was not a good filename. What did you expect here? ;-)

Reply:
I can't link back to anymore, but if you go to Checks, you see this:

![image](https://user-images.githubusercontent.com/155810/115570980-d8d11880-a2be-11eb-98fb-010c444c2e7d.png)

Previously that showed the filename instead of `CI`. Looking at it now, Continuous integration would probably also have been fine since there's sufficient space. I hope that helps to understand why I cared about it.

Reply:
OK, thx for explanations.